packages/web-demo/src/app/bootstrap/components/background/background-demo-page/background-demo-page.component.html

```diff
@@ -11,16 +11,42 @@ <h2 class="bold">Infos</h2>
 </p>
```

Comment:
Warning, Error, Info and Success are contextual backgrounds that are only allowed in Intranet Contexts and should not be part of the main color set.

Can you make 3 sections:

**Default backgrounds**
- Primary
- Yellow
- Light
- Gray
- Dark
- White
- Transparent

**Thematic backgrounds**
- Theme colors (except yellow)

**Contextual backgrounds (Intranet only)**
- Success
- Danger
- Warning
- Info

Reply:
Should we remove the first h2 with reference to bootstrap V5 backgrounds and the button to the bootstrap documentation too? Maybe we should say something like "custom post backgrounds" instead of "Using Bootstrap v5.0".